scripts/config/notebook-testing.toml

```diff
@@ -163,4 +163,11 @@ notebooks = [
     "docs/tutorials/compute-dissociation-curves-for-strong-coupling-systems-with-quna-sys-qsci.ipynb",
     "docs/tutorials/solve-higher-order-binary-optimization-problems-with-q-ctrls-optimization-solver.ipynb",
     "docs/tutorials/chsh-inequality.ipynb",
-]
+
+    # Don't test any learning notebooks
```

Comment:
Thoughts on supporting globs? Maybe adding it as a feature request in GitHub?

Reply:
I think at some point we decided to list each notebook explicitly, but I can't remember why

Reply:
Yeah, because we wanted to avoid unsafe defaults. It would be dangerous to do something like `guides/*` or even `tutorials/*`. 

The difference now is that courses are entirely self-contained, like each course has its own folder. So it would be safe to say `my-course/*`.